doc/image_analysis.md

```diff
@@ -0,0 +1,89 @@
+
+```
+sudo apt install sextractor scamp psfex swarp
+```
+
+Also, one needs to install `snid`. To install it, you can run the following commands which will git clone and run an install script (works only on ubuntu for now, both arm64 and amd64)
+
+```
+git clone https://github.com/Theodlz/snid-install-ubuntu.git && \
+cd snid-install-ubuntu && sudo chmod +x install.sh && sudo bash ./install.sh
+```
+
+## Installing STDpipe
+
+As of right now, STDpipe is in development, and not available on pypi or conda yet.
+STDpipe is available at https://github.com/karpov-sv/stdpipe and is mirrored at https://gitlab.in2p3.fr/icare/stdpipe
+
+1. Get the installer
+
+```
+cd skyportal
+git clone https://github.com/karpov-sv/stdpipe.git
+```
+
+
+2. Installing it in your virtual environment
+
+```
+source skyportal_env/bin/activate
+cd stdpipe
+pip install -e .
+```
+
+## Configuration file
+
+By default, the app allows a maximum request size of 10MB (from the client). This is not enough for someone to upload most .fits files (compressed or not).
+To increase the maximum request size, change the `max_body_size: 10` parameter in the `config.yaml` file to a larger value, like `100` (in MB).
+
+Also, as mentioned earlier, you need to set `image_analysis` to `True` in the `config.yaml` file. By default, it is set to `False`.
+
+Moreover, in the app.routes, uncomment the following lines:
+```
+- path: "/source/:id/image_analysis"
+  component: ImageAnalysisPage
+```
+This will add the frontend route to the image analysis page.
+
+**If you are deploying SkyPortal using Docker, don't forget to add the same changes to the `docker.yaml` file as well.**
+
+## Deploy using Docker
+
+If you are deploying SkyPortal using Docker, you need to uncomment a few lines from the Dockerfile, which are commented out by default. These lines are here to install the system and python dependencies required by STDpipe.
+
+```
+RUN apt-get update && \
+    apt-get install -y sextractor scamp psfex
+```
+
+and
+
+```
+RUN git clone https://github.com/Theodlz/snid-install-ubuntu.git && \
+    cd snid-install-ubuntu && chmod +x install.sh && bash ./install.sh
+
+RUN python3 -m venv /skyportal_env && \
+    bash -c "source /skyportal_env/bin/activate && \
+    git clone https://github.com/karpov-sv/stdpipe.git && \
+    cd stdpipe && pip install -e . && \
+    pip install astroscrappy"
+```
+
+Again, add the same lines mentioned in the Configuration section to the `docker.yaml` file.
+
+## Usage
+
+Go to the page of a source, and add `/image_analysis` at the end of the URL. It should look something like `http://localhost:5000/source/ZTF22aayaehk/image_analysis`.
+Then, fill in the fields required by the form, and upload the image.
+Otherwise, you can find a button labelled `Image Analysis` right under the photometry plot. It will redirect you to the image analysis page.
+
+**The image must be a compressed FITS file (.fits.fz). We recommend compressing it with funpack.**
```

Comment:
Otherwise, it can get way too big for nginx, and it the request doesn't go through. I made sure that the max size was configurable in a PR few weeks ago, but it's still a problem for large images which would require us to allow very large requests (which can be a problem if numerous). 
It's also very easy to compress them, and most libraries (like astropy) can read the compressed file without any problem.